traffic/configure-ios.md

```diff
@@ -15,28 +15,25 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
 To capture and inspect traffic on iOS devices or simulators, perform the following steps:
 
 1. [Provide the prerequisites](#prerequisites).
-1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere).
-1. [Configure the iOS device](#configuring-the-ios-device).
-1. [Configure the iOS simulator](#configuring-the-ios-simulator).
-1. [Inspect the traffic](#inspecting-the-browser-traffic).
+1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere-host).
+1. [Configure the iOS device](#configuring-ios-devices).
+1. [Configure the iOS simulator](#configuring-ios-simulator).
+1. [Inspect the traffic](#inspecting-traffic).
 
 ## Prerequisites
 
 - Install [the latest version of Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
 
 - Use an iOS device or an iOS simulator within the same local network as the Fiddler Everywhere host.
 
-## Configuring Fiddler Everywhere
+## Configuring Fiddler Everywhere Host
```

Comment:
We must note that Fiddler's root certificate must be trusted and HTTPS capturing to be enabled on the machine where Fiddler Everywhere is running. We are going to remove this limitation in the future, but we have it for the moment.